frontend/src/data/hooks/useOnboarding.ts

```
@@ -37,33 +76,344 @@ export const useOnboarding = () => {
+      case 'companyName':
+        if (!value.trim()) {
+          error = 'Company name is required'
+        } else if (value.length < 2) {
+          error = 'Company name must be at least 2 characters'
+        } else if (value.length > 255) {
+          error = 'Company name must be less than 255 characters'
+        }
+        break
+
+      case 'companyDomain':
+        const domainValidationResult = domainValidationUtils.validateFormat(value)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)